Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Usamos la imagen base elegida
+FROM node:lts-alpine
+
+# Creamos el directorio de trabajo y le damos permiso al usuario de node
+RUN mkdir -p /home/app && chown -R node:node /home/app
+WORKDIR /home/app
+
+# Copiamos el package.json al directorio principal para poder instalar las dependencias
+COPY package.json ./
```

Comment:
¿Esto lo necesitas cuando acabes de instalar?

Reply:
Cuando acabe de instalar no, pero para poder instalar las dependencias sí que es necesario, ¿no? Deberia borrarlo una vez instaladas?

Reply:
¿Qué fue lo que dijimos en clase?
Dicho de otro modo, ¿qué tiene que estar dentro de la imagen?

Reply:
¿Seguro que necesitas *solo* eso? ¿Para qué sirven los ficheros `lock`?

Reply:
Según tengo entendido, los ficheros lock sirven para organizar las subdependencias. A la hora de instalar dependencias con `pnpm install` simplemente es necesario el package.json, por lo que solo copio este y despues lo elimino tras instalarlas.

Reply:
En general, cuando os pongo una objeción no es para examinaros de algo. Es para que tratéis de investigar por qué os lo estoy preguntando y lo incorporéis. En este caso además, si no usas el fichero lock, ¿para qué lo pones en la construcción del Dockerfile?

Reply:
No entiendo a que se refiere con "en la construcción del Dockerfile". ¿Se refiere a la github action?

Reply:
> No entiendo a que se refiere con "en la construcción del Dockerfile". ¿Se refiere a la github action?

¿Dónde se construye el Dockerfile?